### PR TITLE
docs(changelog) 2.1.3.0

### PR DESCRIPTION
**Pull request by GayleNeumann**
- Draft of 2.1.3.0 changelog

### CODE REVIEW

File: app/enterprise/changelog.md [hunk 1]
```diff
@@ -5,6 +5,128 @@ no_version: true
...
+
+* Updated plugins bundled with Kong Enterprise for 2.1.x compatibility
+
+### Known Issues and Workarounds
+
+**Note: This 2.1.3.0 version includes the [2.1.0.0 (beta)](/enterprise/changelog/#known-issues-and-workarounds) known issues and workarounds.**
+
+* The [Rate Limiting Advanced](/hub/rate-limiting-advanced) plugin does not support the `cluster` strategy in hybrid mode. The `redis` strategy must be used instead.
+
+* The [Key Authentication Encryptions](hub/kong-inc/key-auth-enc/) (`key-auth-enc`) plugin does not function in hybrid deployment mode.
```
Comment (RobSerafini) [inline app/enterprise/changelog.md:115]: The good news is we fixed this. You can remove the key-auth-enc bullet.

Comment (GayleNeumann) [reply app/enterprise/changelog.md:114]: Great -- will do! Thanks!

File: app/enterprise/changelog.md [hunk 2]
```diff
@@ -5,6 +5,127 @@ no_version: true
...
+Kong Enterprise 2.1.3.0 version includes 2.1.0.0 (beta) features, fixes, known issues, and workarounds. See the [2.1.0.0 (beta)](/enterprise/changelog/#2100-beta/) changelog for more details.
+
+### Features
+
+**Note: Feature updates for Kong Enterprise 2.1.3.0 version includes [2.1.0.0 (beta)](/enterprise/changelog/#features) features.**
+
+#### Kong Gateway
+* Inherited changes from OSS Kong in releases 2.0.x, 2.1.0, 2.1.1, 2.1.2, and 2.1.3.
+* Workspaces code has been refactored for performance. The feature should work the same for most users.
+* TLS version may be specified when using tls to connect to a Postgres db.
```
Comment (eskibars) [inline app/enterprise/changelog.md:20]: `db` should be `DB` or `database`.  `tls` should be `TLS`

File: app/enterprise/changelog.md [hunk 3]
```diff
@@ -5,6 +5,127 @@ no_version: true
...
+
+#### Kong Manager
+* Open ID Connect (`openid-connect`) can now be used with [Mapping Service Directory Groups to Kong Roles](/enterprise/latest/kong-manager/service-directory-mapping/) using `config.authenticated_groups_claim`.
+* Can now see the `created_at` timestamp for consumer credentials.
+* Provide a warning when a user's session is about to expire.
+* Generate an RBAC token for an administrator rather than letting them set one.
+* Plugin forms now have docs links to the Plugins Hub.
```
Comment (eskibars) [inline app/enterprise/changelog.md:27]: Does it make sense to have a link to `/hub` for the Plugins Hub?

File: app/enterprise/changelog.md [hunk 4]
```diff
@@ -5,6 +5,127 @@ no_version: true
...
+
+### Fixes
+
+**Note: Fixes for Kong Enterprise 2.1.3.0 version includes the [2.1.0.0 (beta)](/enterprise/changelog/#fixes) fixes.**
+
+#### Kong Manager
+* Auto-generated role `workspace-portal-admin` does not display link to Dev Portal Editor.
+* Routes form did not expose `path_handling` attribute for adding/editing.
+* When Kong Manager is secured with OIDC, new workspaces do not include the auto-generated roles.
+* Able to enumerate usernames via the /admins endpoint.
```
Comment (eskibars) [inline app/enterprise/changelog.md:64]: Can we put `/admins` in backticks?

File: app/enterprise/changelog.md (hunk 2 above)
Comment (GayleNeumann) [reply app/enterprise/changelog.md:20]: Fixed.

File: app/enterprise/changelog.md [hunk 5]
```diff
@@ -5,6 +5,127 @@ no_version: true
...
+
+#### Kong Developer Portal
+* The `/developers/:developer/applications` endpoint allowed applications to be created for a developer other than the one in the path.
+* The change password endpoint for developers did not require the original password.
+* User was not redirected to Login after a session timeout.
+* Unable to configure `Accept Http If Already Terminated` for Application Registration plugin. Instead, add both Application Registration and OAuth 2.0 Authentication, and set the config on the Oauth2 Authentication plugin.
+
+#### Plugins
+
+* [gRPC Gateway](/hub/kong-inc/grpc-gateway/)documentation is improved.
```
[the remark this answers is not in the document]
Comment (GayleNeumann) [reply app/enterprise/changelog.md:75]: fixed.

File: app/enterprise/changelog.md (hunk 4 above)
Comment (GayleNeumann) [reply app/enterprise/changelog.md:64]: fixed.

File: app/enterprise/changelog.md (hunk 3 above)
Comment (GayleNeumann) [reply app/enterprise/changelog.md:27]: yes. done.

**Pull request merged**